core/src/main/java/org/frankframework/xml/SaxElementBuilder.java

```diff
@@ -126,12 +126,14 @@ public void addValue(char[] chars, int offset, int len) throws SAXException {
 	}
 
 	public SaxElementBuilder startElement(String elementName) throws SAXException {
-		if (elementName==null) {
+		String cleanElementName = XmlUtils.cleanseElementName(elementName);
```

Comment:
We already had a method to 'cleanse' an xml element name. The unit test proves that this works as expected.

Reply:
Kan dit ook in de XmlBuilder en SaxDocumentBuilder?

Reply:
Ik heb dit aangepast in de XmlBuilder en in SaxElementBuilder constructor, daar komt de SaxDocumentBuilder namelijk altijd langs. Ik denk dat dat de 3 scenario's dekt. Mee eens?